backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -26,11 +27,13 @@ import { LoginUserRepository } from '../../typeorm/repository/LoginUser'
 @Resolver()
 export class AdminResolver {
   @Authorized([RIGHTS.SEARCH_USERS])
-  @Query(() => [UserAdmin])
-  async searchUsers(@Arg('searchText') searchText: string): Promise<UserAdmin[]> {
+  @Query(() => SearchUsersResult)
+  async searchUsers(
+    @Args() { searchText, currentPage = 1, pageSize = 25, notActivated = false }: SearchUsersArgs,
+  ): Promise<SearchUsersResult> {
     const userRepository = getCustomRepository(UserRepository)
     const users = await userRepository.findBySearchCriteria(searchText)
-    const adminUsers = await Promise.all(
+    let adminUsers = await Promise.all(
```

Comment:
I see whats happening here - this is ugly - please add a todo - we MUST refactor this